HitDelay.py

```
@@ -1,429 +1,439 @@
+				data.pop(-1);
+			for ids in range(1,len(data)): # 去除第一行
+				dataList.append(float(data[ids][13:-3]));
+			allKeys = len(dataList);
+			sumNums,sumKeys = 0,0;
+			for ids in dataList:
+				if (ids < self.delayInterval) and (ids > -self.delayInterval):
+					sumNums += ids;
+					sumKeys += 1;
+			avgDelay = (sumNums / sumKeys) if sumKeys > 0 else 0
+			avgAcc = (sum(abs(i) for i in dataList) / allKeys) if allKeys > 0 else 0
+			self.delayHistory.insert('', END, values=(name,allKeys,'%.6f ms'%avgDelay,'%.6f ms'%avgAcc));
```

Ruff (0.14.8)

`END` may be undefined, or defined from star imports

(F405)